egs/multi_en/s5/local/make_partitions.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2016  Allen Guo
+# Apache License 2.0
+
```

Comment:
Thanks for the feedback! Addressed this and some of your other in-line comments in 06ac0e0f902f7a01eb4861e8a331a22edec0192f.